kubenow/pachyderm/README.md

```diff
@@ -1,28 +1,27 @@
 Pachyderm Helm Chart
```

Comment:
please remove these changes. So we'll push this to the official k8s, and there we can assume that the users know how to provision S3 on their own. Also, for AWS deployments they don't really need the S3 dep.